src/main/kotlin/no/skatteetaten/aurora/boober/mapper/platform/JavaPlatformHandler.kt

```diff
@@ -0,0 +1,61 @@
+package no.skatteetaten.aurora.boober.mapper.platform
+
+import no.skatteetaten.aurora.boober.mapper.AuroraConfigFieldHandler
+import no.skatteetaten.aurora.boober.model.AuroraDeploymentSpec
+import no.skatteetaten.aurora.boober.model.Mount
+import no.skatteetaten.aurora.boober.model.TemplateType.development
+import no.skatteetaten.aurora.boober.platform.ApplicationPlatformHandler
+import no.skatteetaten.aurora.boober.platform.AuroraContainer
+import no.skatteetaten.aurora.boober.platform.AuroraDeployment
+import no.skatteetaten.aurora.boober.utils.addIfNotNull
+import org.springframework.stereotype.Component
+
+@Component
+class JavaPlatformHandler : ApplicationPlatformHandler("java") {
```

Comment:
Should these handler implementations have unit tests? Or are they perhaps covered by other tests?

Reply:
They are implicitly tested with other tests, I am not sure how much value it would give to test them separately. 

They are tested in the OpenShiftObjectGeneratorTest. However it would be very nice to refactor these tests. I feel that right now that is too big a task for this PR. It is a task of its own to refactor that test into smaller tests.